### PR TITLE
fix grammar and other things in the osu!tourney article

### PR DESCRIPTION
**Pull request by oralekin**
should be squashed when merging bcs im bad at committing

### CODE REVIEW

File: wiki/osu!tourney/en.md [hunk 1]
```diff
@@ -296,14 +292,13 @@ We may award badges to the first place winners of second (and further) iteration
 
 ## Troubleshooting
 
-### How do fresh install osu! without uninstalling my current?
+### How do I fresh install osu! without uninstalling my current?
 
-Copy-paste the `osu!.exe` into an empty folder and run it.
+Copy the `osu!.exe` into an empty folder and run it.
```
Comment (smoogipooo) [inline wiki/osu!tourney/en.md]: Probably should be "Copy `osu!.exe` into [...]".

File: wiki/osu!tourney/en.md [hunk 2]
```diff
@@ -296,14 +292,13 @@ We may award badges to the first place winners of second (and further) iteration
 
 ## Troubleshooting
 
-### How do fresh install osu! without uninstalling my current?
+### How do I fresh install osu! without uninstalling my current?
```
Comment (smoogipooo) [inline wiki/osu!tourney/en.md]: "How do I create a fresh osu! installation without uninstalling the current game?"

File: wiki/osu!tourney/en.md [hunk 3]
```diff
@@ -312,10 +307,10 @@ Make sure to have osu! updated to the newest cuttingedge build!
...
 ### osu!tourney does not open, it throws me an error prompt and/or closes!
 
-- Ensure that osu! is using the Cutting Edge release stream.
-- Make sure that you are logged into osu!
-  - If not, run osu! normally, login while making sure the "Remember Username" and "Remember Password" checkboxes are checked, then restart osu! as the tournament client.
-- Ensure that the `privateserver` key in the `tournament.cfg` has no value set.
+Ensure that osu! is using the Cutting Edge release stream.
+Make sure that you are logged into osu!
+If not, run osu! normally, login while making sure the "Remember Username" and "Remember Password" checkboxes are checked, then restart osu! as the tournament client.
+Ensure that the `privateserver` key in the `tournament.cfg` has no value set.
```
Comment (smoogipooo) [inline wiki/osu!tourney/en.md]: I wanted these to be dot points. Sort of like steps you check.

File: wiki/osu!tourney/en.md [hunk 4]
```diff
@@ -332,34 +327,33 @@ Rename or delete the `tournament.cfg` file.
...
 Click the room name on the bottom black panel until it is **bold**.
 If the client is still not spectating, click the `Panic` button.
 
 ### The rooms are not showing up!
 
-- Your multiplayer room is named incorrectly or you used the wrong acronym in the `tournament.cfg`.
-- Make sure you use the correct room name (see the **Match Creation** section).
-  Renaming an existing multiplayer room will not work.
-- Recreate the entire room, if you did something wrong.
+It is possible that your multiplayer room is named incorrectly or you used the wrong acronym in the `tournament.cfg`.
```
Comment (smoogipooo) [inline wiki/osu!tourney/en.md]: "in `tournament.cfg`."

File: wiki/osu!tourney/en.md [hunk 5]
```diff
@@ -332,34 +327,33 @@ Rename or delete the `tournament.cfg` file.
...
+Make sure you are using the correct room name (see the **Match Creation** section), renaming an existing multiplayer room will not work.
```
Comment (smoogipooo) [inline wiki/osu!tourney/en.md]: Replace comma with hyphen I think.

File: wiki/osu!tourney/en.md [hunk 6]
```diff
@@ -332,34 +327,33 @@ Rename or delete the `tournament.cfg` file.
...
+Recreate the entire room if you did something wrong.
```
Comment (smoogipooo) [inline wiki/osu!tourney/en.md]: Would reword this to "If the above doesn't work recreate the room while following the **Match Creation** section of this guide closely."

File: wiki/osu!tourney/en.md [hunk 7]
```diff
@@ -332,34 +327,33 @@ Rename or delete the `tournament.cfg` file.
...
+
+Post in [this thread](https://osu.ppy.sh/forum/t/327120) if you need further assistance.
 
 ### How do I stream my tournament to streaming services like Twitch?
 
 To set up a stream, we recommend using the [Open Broadcaster Software](https://obsproject.com/), but know that you are also free to use any other streaming client you have access to.
-The resolution of the top part of osu!tourney is `1280x720px`.
-
-**Please adjust the cropping to remove the black control panel from the stream!**
+The resolution of the top part of osu!tourney can be customised but by default it is `1280 × 720`. Make sure yoir monitor is bigger than this value. See the Setup section for information on how to change the resolution.
```
Comment (smoogipooo) [inline wiki/osu!tourney/en.md]: your monitor.*

File: wiki/osu!tourney/en.md [hunk 8]
```diff
@@ -332,34 +327,33 @@ Rename or delete the `tournament.cfg` file.
...
 
 ### The team names are not showing!
 
-- Your multiplayer room is named incorrectly then or you use the wrong acronym in the `tournament.cfg`.
-- Make sure you use the correct room name (see the **Match Creation** section).
-  Renaming an existing multiplayer room will not work.
-- Recreate the entire room, if you did something wrong.
+It is possible that your multiplayer room is named incorrectly or you used the wrong acronym in the `tournament.cfg`.
+Make sure you use the correct room name (see the **Match Creation** section), renaming an existing multiplayer room will not work.
+Recreate the entire room if you did something wrong.
```
Comment (smoogipooo) [inline wiki/osu!tourney/en.md]: Can we not duplicate this? Maybe link it back to `The rooms are not showing up!`?

File: wiki/osu!tourney/en.md [hunk 9]
```diff
@@ -332,34 +327,33 @@ Rename or delete the `tournament.cfg` file.
...
+
+Post in [this thread](https://osu.ppy.sh/forum/t/327120) if you need further assistance.
```
Comment (smoogipooo) [inline wiki/osu!tourney/en.md]: Don't link to forum threads. Should be "Email `tournaments@ppy.sh` if you require further assistance.".

File: wiki/osu!tourney/en.md [hunk 10]
```diff
@@ -332,34 +327,33 @@ Rename or delete the `tournament.cfg` file.
...
 To set up a stream, we recommend using the [Open Broadcaster Software](https://obsproject.com/), but know that you are also free to use any other streaming client you have access to.
-The resolution of the top part of osu!tourney is `1280x720px`.
-
-**Please adjust the cropping to remove the black control panel from the stream!**
+The resolution of the top part of osu!tourney can be customised but by default it is `1280 × 720`. Make sure yoir monitor is bigger than this value. See the Setup section for information on how to change the resolution.
+Do not forget to adjust the cropping to remove the black control panel from the stream!
 
 ### My issue/question is not listed here! What to do?
 
 Post in **[this thread](https://osu.ppy.sh/forum/t/327120)**.
```
Comment (smoogipooo) [inline wiki/osu!tourney/en.md]: This needs to be removed and should link back to `tournaments@ppy.sh`. The line below this one also, but keep the screenshots one.

File: wiki/osu!tourney/en.md [hunk 11]
```diff
@@ -296,14 +292,13 @@ We may award badges to the first place winners of second (and further) iteration
...
-Copy-paste the `osu!.exe` into an empty folder and run it.
+Copy the `osu!.exe` into an empty folder and run it.
 
 ### My osu! windows are not aligning properly!
 
-**Please disable any secondary monitors!**
-
+Disable any **secondary monitors!**
 osu!tourney only runs on the primary monitor.
 A small screen resolution could also cause this problem as well.
```
Comment (smoogipooo) [inline wiki/osu!tourney/en.md]: Let's reword this one to "osu!tourney only runs on the primary monitor. Try disabling any secondary monitors you have active. Make sure that the primary monitor has a resolution larger than the resolution of the client (1280x720 by default).

File: wiki/osu!tourney/en.md (hunk 5 above)
Comment (oralekin) [inline wiki/osu!tourney/en.md]: A hyphen would.l serve the same purpose as the comma here but i prefer the comma personally.

Comment (smoogipooo) [inline wiki/osu!tourney/en.md]: Alright.
